README.md

```diff
@@ -12,9 +12,9 @@ If you have logged in with an administrator account, you will be able to manage
 Contributors are welcome. If you want to help, feel free to submit a pull request or open an issue.
 
 #### Code Style
-4 space tabs. No tab characters, no 2 space tabs, no 8 space tabs.<br>
-Max line length is 120 characters, but preferably keep it below 100.<br>
-Make it readable. Use descriptive variable names, and define them close to the first use.
+Follow the [PEP 8](https://www.python.org/dev/peps/pep-0008/) style guide for Python code.
+
+For non-Python code, please use 4 space tabs, limit your lines to under 120 characters, and use descriptive variable names.
```

Comment:
Change to under 100 characters

Reply:
I also want to change the Python line limit to 90. I know PEP 8 specifies 79 but that's frankly annoying sometimes (and they say some people do make it longer, up to a maximum of 99). Even with 90, it's short enough that if someone were to split screen, code still wouldn't spill past the edge of the window